metadata/PricingRecipe/pricingRecipe/DigitalLendingRecipe.pricingRecipe

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<PricingRecipe xmlns="http://soap.sforce.com/2006/04/metadata">
+    <defaultPricingProcedureId> </defaultPricingProcedureId>
+    <developerName>DigitalLendingRecipe</developerName>
+    <isActive>true</isActive>
+    <isInternal>false</isInternal>
+    <masterLabel>DigitalLendingRecipe</masterLabel>
+    <pricingRecipeTableMapping>
+        <isInternal>false</isInternal>
+        <lookupTableDeveloperName>CreditScoreBasedAdjustment</lookupTableDeveloperName>
+        <pricingComponentType>CUSTOMDISCOUNT</pricingComponentType>
+        <pricingProcedureOutputMapList>
+            <fieldName>TierValue__c</fieldName>
+            <isPricingRecipeActive>false</isPricingRecipeActive>
+            <outputFieldName>0lPxx000000006U</outputFieldName>
+            <outputType>AdjustmentValue</outputType>
+        </pricingProcedureOutputMapList>
+        <pricingProcedureOutputMapList>
+            <fieldName>TierType__c</fieldName>
+            <isPricingRecipeActive>false</isPricingRecipeActive>
+            <outputFieldName>0lPxx000000006W</outputFieldName>
+            <outputType>AdjustmentType</outputType>
+        </pricingProcedureOutputMapList>
+        <pricingRecipe>DigitalLendingRecipe</pricingRecipe>
+    </pricingRecipeTableMapping>
+    <pricingRecipeTableMapping>
+        <isInternal>false</isInternal>
+        <lookupTableDeveloperName>ProductListRateDT</lookupTableDeveloperName>
+        <pricingComponentType>LISTPRICE</pricingComponentType>
+        <pricingProcedureOutputMapList>
+            <fieldName>ListRate</fieldName>
+            <isPricingRecipeActive>false</isPricingRecipeActive>
+            <outputFieldName>0lPxx000000006Y</outputFieldName>
+            <outputType>UnitPrice</outputType>
+        </pricingProcedureOutputMapList>
+        <pricingRecipe>DigitalLendingRecipe</pricingRecipe>
+    </pricingRecipeTableMapping>
+    <pricingRecipeTableMapping>
+        <isInternal>false</isInternal>
+        <lookupTableDeveloperName>TermBasedAdjustmentDT</lookupTableDeveloperName>
+        <pricingComponentType>CUSTOMDISCOUNT</pricingComponentType>
+        <pricingProcedureOutputMapList>
+            <fieldName>TierValue__c</fieldName>
+            <isPricingRecipeActive>false</isPricingRecipeActive>
+            <outputFieldName>0lPxx000000006a</outputFieldName>
```

Comment:
seems like an id, does this need to be replaced or it doesn't matter?

Reply:
@mitsenthil Yes its an id but doesnt matter if its present in the org to be deployed and does not fail. So its safe to have it here as the retrieve file came with this id

Reply:
Where do we hardcode that id that we want to create the file with?